ilc/client/TransitionManager/SlotRenderObserver/SlotRenderObserver.js

```diff
@@ -0,0 +1,53 @@
+export class SlotRenderObserver {
+
+    #observer;
+
+    observe(targetNode, { onSlotReady }) {
+
+        if(this.#observer) {
```

Comment:
there is a `disconnect` method, so I assume flow

```js
const observer  = new SlotRendererObserver()
observer.observe();
observer.disconnect();
observer.observe();
```

is correct, but `disconnect` leaves disconnected `#observer` instance, so it will not work right now

Reply:
I mean, that `disconnect` technically does not allow right now to run new `observe`. So the instance is usable one time.